### PR TITLE
Add R/Rserve

### PR DESCRIPTION
**Pull request by artemklevtsov**

### CODE REVIEW

**Comment by waghanza** [comment]
Thanks @artemklevtsov.

@s-u do you consent, at least not opposed, to include https://github.com/s-u/Rserve in `here` ?

**Comment by s-u** [comment]
@waghanza yes, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -136,6 +136,7 @@ env:
       - FRAMEWORK=java.light-4j
       - FRAMEWORK=r.httpuv
       - FRAMEWORK=r.plumber
+      - FRAMEWORK=r.rserve
       - FRAMEWORK=r.restrserve
       - FRAMEWORK=fsharp.falco
       - FRAMEWORK=fsharp.suave
```
Comment (waghanza) [inline .travis.yml:142]: could you remove this file ? no longer required with the new CI, but please you need to run `rake ci:config` before push

File: r/rserve/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: rforge.net/Rserve/
+  version: 1.7-3
```
Comment (waghanza) [inline r/rserve/config.yaml:3]: could you use `X.Y` format ?

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by t-wojciech** [comment]
@s-u There are already results from this benchmark and `Rserve` is doing quite well.

> Pos  | Language | Framework | Speed (64) | Speed (256) | Speed (512)
> 107 | r (4.0) | rserve (1.7) | 41 968 | 29 638 | 29 848

**Pull request merged**